pmemkv/src/test/resources/config_params

```diff
@@ -0,0 +1 @@
+{"param1": "abc", "param2": 123, "create_if_missing": true}
```

Comment:
maybe add .json to the file name?

Reply:
create_if_missing requires pmemkv 1.5.0, is this expected? if so, we may as well replace the logic in PmemkvClient and use create_if_missing probably.

Reply:
1. Renamed.
2. oops, nope this was just for my testing - as you can see I put there some fake fields. pmemkv-java 1.2.0 requires pmemkv 1.4 so, I wouldn't change it here (at least for now). Removed the flag.